internal/provisioner/kops_provisioner_cluster.go

```diff
@@ -190,18 +190,6 @@ func (provisioner *KopsProvisioner) CreateCluster(cluster *model.Cluster, awsCli
 
 	logger.WithField("name", kopsMetadata.Name).Info("Successfully deployed kubernetes")
 
-	logger.WithField("name", kopsMetadata.Name).Info("Updating VolumeBindingMode in default storage class")
-	k8sClient, err := k8s.NewFromFile(kops.GetKubeConfigPath(), logger)
-	if err != nil {
-		return err
-	}
-
-	_, err = k8sClient.UpdateStorageClassVolumeBindingMode("gp2")
-	if err != nil {
-		return err
-	}
-	logger.WithField("name", kopsMetadata.Name).Info("Successfully updated storage class")
```

Comment:
This block fails at the moment and in my understanding of things is totally obsolete and can be safely removed. But if your understanding is different @gabrieljackson and @Szymongib feel free to correct me.

Reply:
Hmm, I can't remember what this was used for, but would agree that if there are no perceived issues then it should be okay to remove. The code was added by @stylianosrigas so let's check with him as well.